src/app/app.component.ts

```diff
@@ -13,16 +12,17 @@ export class AppComponent {
     public position = 'col-md-10 col-md-offset-2';
     public loggedIn = false;
 
-    constructor(private _router: Router, private _sharedData: SharedDataService) {
+    constructor(private _router: Router) {
         this._router.events.pairwise().subscribe((e) => {
             this.loggedIn = (e && localStorage.getItem(AppComponent._localStorageKey)) ? true : false;
+            this.position = (this.loggedIn === true) ? 'col-md-10 col-md-offset-2' : 'col-md-12';
         });
     }
 
     /**
-     * Call toggle function in shared data service
+     * Call toggle function
      */
-    toggle() {
-        this._sharedData.toggle();
-    }
+     toggle() {
```

Comment:
the app.component.ts is a base HTML for all others , we created sharedData.toggle() to remove many toggles, but now that we have a base class we just need one toggle function in the base HTML file. It is not needed anywhere else. That is why I discussed with Rajat and removed sharedData.toggle()  using the normal toggle function was a neat and simpler way.